src/charts/stacked-bar.js

```diff
@@ -0,0 +1,1010 @@
+define(function(require){
+    'use strict';
+
+     const d3Array = require('d3-array'),
+     d3Axis = require('d3-axis'),
```

Comment:
ok,I just followed vscode tip,It highlight this block,so I combined const declares.I will keep watch the code style guide progress.